examples/rp-process/rp_process.py

```diff
@@ -148,7 +148,7 @@ def limit_products(r):
     encountered.update(prod)
     
 encountered = sorted(encountered)
```

Comment:
Running `./rp_process.py te108 --write_lib rp-process-lib` for the reduction code fails with a `RateDuplicationError`; this fixes it.
```suggestion
encountered = sorted(encountered)

# remove duplicate rates
dupes = final_lib.find_duplicate_links()
for d in dupes:
    # if wc17 is present, then remove all other duplicate rates
    labels = {r.label: r for r in d}
    if len(labels) == len(d) and "wc17" in labels:
        for r in d:
            if r.label == "wc17":
                continue
            final_lib.remove_rate(r)
        r = labels["wc17"]
        print(f"Found rate {r} named {r.fname} with {len(d)} entries in the Library.")
        print(f"Kept only entry with label {r.label} out of {list(labels.keys())}.")

```